navigator:
  arm_group: ur5_arm
  gripper_group: ur5_gripper
  wrist_link: ur5_arm_ee_link
  gripper_link: ur5_arm_tcp_frame
  planner_service_name: /ur5_arm/path_planning/plan_kinematic_path
  trajectory_filter_service_name: /ur5_arm/trajectory_filter_server/filter_trajectory_with_constraints
  segmentation_service_name: /ur5_arm/tabletop_segmentation
  recognition_service_name: /ur5_arm/mantis_object_recognition
  grasp_planning_service_name: /ur5_arm/plan_point_cluster_grasp
  planning_scene_service_name: /ur5_arm/environment_server/set_planning_scene_diff
  arm_inverse_kinematics_plugin: arm_kinematics_constraint_aware/KDLArmKinematicsPlugin
  joint_state_topic: /ur5_arm/joint_states
  joint_trajectory_service_name: /ur5_arm/joint_trajectory_action
  grasp_action_service_name: /ur5_arm/grasp_execution_action
segmentation:
  world_frame_id: workcell_frame
  k_nearest: 40
  max_iterations: 400
  sac_method: 6 #ransac = 0
  normal_distance_weight: 0.2
  distance_threshold: 0.0005
  max_radius: 0.028
  min_fitness_score: 20 # minimum number of inliers
  align_to_top_centroid: true
goal:
  frame_id: ur5_arm_base_link
  position:
    x: 0.451
    y: -0.493
    z: 0.2
  orientation:
    axis:
      x: 0
      y: 0
      z: 1
    angle: 0
  next_location:
    generation_mode: 0
    min_spacing: 0.05 # meters
    max_spacing: 0.08 # meters
    place_region_radius: 0.2 # meters
joints:
  side_position: [-1.51, -1.03, 1.48, -2.03, -1.55, -0.17]
